buildkite/bazel-bench/bazel_bench.py

```diff
@@ -65,40 +62,42 @@ def _bazel_bench_env_setup_command(platform, bazel_commits):
     return [download_command, exec_command]
 
 
-def _get_bazel_commits(day, bazel_repo_path):
-    """Get the commits from a particular day.
+def _get_bazel_commits(date, bazel_repo_path):
+    """Get the commits from a particular date.
```

Comment:
I know it's generic but it would read a lot nicer with today/tomorrow instead of date and date_plus_one.

Reply:
Acknowledged. However, I find it quite restrictive to nail the dates down as "today/tomorrow" when in fact we support any date.